test/rocprim/test_utils_types.hpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2017 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Copyright 2019?

Reply:
Shouldn't it be 2017-2019?

Reply:
That seems to be a new file, so 2019 might work. Please merge after any correction that you find acceptable.

Reply:
Fixed